scripts/config/lib/ngx_mruby/locations_state_path.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal=> true
+
+# ghetto require, since mruby doesn't have require
+eval(File.read('/app/bin/config/lib/nginx_config_util.rb'))
+
+USER_CONFIG = '/app/static.json'
+
+req         = Nginx::Request.new
+uri         = req.var.uri
+
+def updated_state(abbr_state)
+    state_map = {
```

Comment:
I use a function to check abbreviated state names, so when the input is not valid we won't throw an error.
Since we are using the hash in this function only, I decided to just leave it in the method instead of creating a global variable.